### ISSUE TITLE
Class Google\Task\Composer is not autoloadable

### ISSUE THREAD

**Issue by cacharrin** [googleapis/google-api-php-client#1971]
#### Environment details

  - OS: Windows 10
  - PHP version: 7.4
  - Package name and version: 2.7

#### Explanation

Hi, I'm trying to use the library just to implement "Login with Google" in PHP ( I dont need any other google service).

I tried to configure the post update CMD like documentation says in its basic example:

```json
{
    "require": {
        "google/apiclient": "^2.7"
    },
    "scripts": {
        "post-update-cmd": "Google\\Task\\Composer::cleanup"
    },
    "extra": {
        "google/apiclient-services": []
    }
}
```

Whe in run `composer update`, after dependencies are installed, it display:

```
Class Google\Task\Composer is not autoloadable, can not call post-update-cmd script
``` 

Do I need to do something else to make `post-update-cmd` command to work?

Thank you!

**Comment by bshaffer** [comment]
@cacharrin thank you for bringing this to our attention! this is because we need to release a new version with the namespaces that were added in #1937

**Comment by bshaffer** [comment]
We've released [v2.8.0](https://github.com/googleapis/google-api-php-client/releases/tag/v2.8.0)! Please update to this version and you'll be good to go!

**Comment by cacharrin** [comment]
Thank you :)
Worked like a charm.
